tests/utilities/fake_driver.py

```diff
@@ -17,34 +17,11 @@
 
 GRPC_SERVICE_INTERFACE_NAME = "nifake_grpc.NiFake"
 
-# The GrpcSessionOptions classes in nimi-python and nidaqmx-python have an api_key field.
```

Comment:
Team, I've removed the below constant (`_API_KEY`) and classes (`SessionInitializationBehavior` & `GrpcSessionOptions`) as they're not used anywhere. 

Please let me know if these were intended for any future purpose.

Reply:
They are intended to model a gRPC-aware driver API. Put them back.